modules/cnf-image-based-upgrade-installing-lifecycle-agent-using-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lifecycle-agent-using-web-console_{context}"]
+= Installing the {lcao} by using the web console
+
+You can use the {product-title} web console to install the {lcao}.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* → *OperatorHub*.
+. Search for the *{lcao}* from the list of available Operators, and then click *Install*.
+. On the *Install Operator* page, under *A specific namespace on the cluster* select *openshift-lifecycle-agent*.
+. Click *Install*.
+
+.Verification
+
+To confirm that the installation is successful:
```

Comment:
Should not have text outside of list items in steps. Could make this a `.` step and the steps under it `..` steps, then do the same with "If the Operator is not installed successfully:" (though really that second one could be under a troubleshooting heading).